test/common/http/http2/codec_impl_test.cc

```diff
@@ -1076,14 +1076,14 @@ TEST_P(Http2CodecImplTest, ShouldDumpActiveStreamsWithoutAllocatingMemory) {
     Stats::TestUtil::MemoryTest memory_test;
     server_->dumpState(ostream, 1);
     EXPECT_EQ(memory_test.consumedBytes(), 0);
-
-    // Check contents for active stream, trailers to encode and header map.
+    // Check contents for active stream, local_end_stream_, trailers to encode and header map.
     EXPECT_THAT(
         ostream.contents(),
         HasSubstr(
             "Number of active streams: 1, current_stream_id_: null Dumping 1 Active Streams:\n"
             "  stream: \n"
             "    ConnectionImpl::StreamImpl"));
+    EXPECT_THAT(ostream.contents(), HasSubstr("local_end_stream_: 1"));
```

Comment:
Should I do additional stream state checks here e.g. check all bools dumped? Don't want it to be too brittle though, which is why I only originally checked it started dumping the stream. WDYT?

Reply:
I think this is sufficient.